docs/app/pages/components/components-sections/tables/column-sorting/column-sorting.component.html

```diff
@@ -1,78 +1,161 @@
+                        [attr.aria-sort]="dateColumn.state"
+                        (click)="changeState('Date Modified', dateColumn)"
+                        (keydown.enter)="changeState('Date Modified', dateColumn)">
+
+                        <div class="column-sorting-header">
+                            <p class="text-black">DATE MODIFIED</p>
+                            <ux-column-sorting #dateColumn="ux-column-sorting" key="date"></ux-column-sorting>
+                        </div>
+                    </th>
+                    <th class="clickable column-sorting"
+                        uxFocusIndicator
+                        tabindex="0"
+                        [attr.aria-label]="getColumnAriaLabel('Work Completed', completedColumn)"
+                        [attr.aria-sort]="completedColumn.state"
+                        (click)="changeState('Work Completed', completedColumn)"
+                        (keydown.enter)="changeState('Work Completed', completedColumn)">
+
+                        <div class="column-sorting-header">
+                            <p class="text-black">WORK COMPLETED</p>
+                            <ux-column-sorting #completedColumn="ux-column-sorting" key="completed"></ux-column-sorting>
+                        </div>
+                    </th>
+                    <th class="text-black">STATUS</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="clickable" *ngFor="let item of items">
+                    <td>{{ item.name }}</td>
+                    <td>{{ item.author }}</td>
+                    <td>{{ item.date }}</td>
+                    <td>
+                        <ux-spark [trackColor]="sparkTrackColor" [barColor]="sparkBarColor" [value]="item.completed" [barHeight]="3"
+                                  [inlineLabel]="item.completed"></ux-spark>
+                    </td>
+                    <td class="text-center">
+                        <ux-icon [name]="item.active ? 'checkmark' : null"
+                                 [attr.aria-label]="item.active ? 'Approved' : 'Not approved'">
+                        </ux-icon>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </ng-container>
+    <ng-container *ngSwitchCase="SiteThemeId.MicroFocus2020">
+        <table class="table table-hover">
+            <thead>
+                <tr uxColumnSorting>
+                    <th class="clickable column-sorting"
+                        uxFocusIndicator
+                        tabindex="0"
+                        [attr.aria-label]="getColumnAriaLabel('Name', nameColumn)"
+                        [attr.aria-sort]="nameColumn.state"
+                        (click)="changeState('Name', nameColumn)"
+                        (keydown.enter)="changeState('Name', nameColumn)">
+
+                        <div class="column-sorting-header">
+                            <p class="text-black">Name</p>
```

Comment:
I think this wrapper around the header text is breaking our 2020 styling:
![image](https://user-images.githubusercontent.com/26114101/97312906-efda0500-185d-11eb-91ce-3b321642f759.png)

The truncation behavior which I mentioned to you previously will not work with this layout:
![image](https://user-images.githubusercontent.com/26114101/97313247-52330580-185e-11eb-9d8c-18260fe76884.png)

Try removing the parent div and just put the heading text in a plain span, like this plunker:
https://plnkr.co/edit/1u4m0fPw4jw3qUPP?preview